packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -43,14 +44,16 @@ dependencies: {{#enable_dev_menu}}
   shelf: ^1.4.1 # This package is used for the local server (if not used, can be removed)
   shelf_router: ^1.1.4 # This package is used for the local server (if not used, can be removed)
   shelf_static: ^1.1.2 # This package is used for the local server (if not used, can be removed){{#enable_social_logins}}
-  sign_in_with_apple: ^5.0.0{{/enable_social_logins}}
+  sign_in_with_apple: ^5.0.0{{/enable_social_logins}}{{#enable_auth_matrix}}
+  uuid: ^3.0.7{{/enable_auth_matrix}}
   widget_toolkit: ^0.0.2-dev.1 {{#enable_pin_code}}
   widget_toolkit_biometrics: ^0.0.2-dev.1 {{/enable_pin_code}} {{#enable_feature_otp}}
   widget_toolkit_otp: ^0.0.2-dev.1{{/enable_feature_otp}}{{#enable_pin_code}}
```

Comment:
Can you synchronize the branch with `develop` and update the `widget_toolkit_otp` package version?